web/gds-admin-ui/src/redux/selectors/vasp-details.js

```diff
@@ -6,4 +6,4 @@ export const getVaspDetails = createSelector(vaspDetailsState, state => state.da
 export const getVaspDetailsLoadingState = createSelector(vaspDetailsState, state => state.loading)
 export const getVaspDetailsErrorState = createSelector(vaspDetailsState, state => state.error)
 export const getTrisaDetailsErrorState = createSelector(vaspDetailsState, state => state.trisaError)
-export const getIvmsErrorState = createSelector(vaspDetailsState, state => state.ivmsError)
\ No newline at end of file
+export const getIvmsErrorState = createSelector(vaspDetailsState, state => state.ivmsError)
```

Comment:
I honestly stared at these two lines of code during the rebase and I can't see what the difference is between them. Just the newline I guess?

Reply:
Perhaps it is due to rebasing and conflits resolving